app.js

```diff
@@ -111,16 +117,79 @@ function setSource(list, num){
 
 function getResults() {
 
-  var resultsContainer = document.getElementById('results-container');
-  var resultsUl = document.createElement('ul');
-  resultsUl.setAttribute('id', 'results-ul');
-  var resultsLi;
-  for(var i = 0; i < products.length; i++) {
-    resultsLi = document.createElement('li');
-    resultsLi.textContent = products[i].timesChosen + ' votes for ' + products[i].name;
-    resultsUl.appendChild(resultsLi);
-  }
-  resultsContainer.appendChild(resultsUl);
+  var ctx = document.getElementById('results-chart');
+  var resultsChart = new Chart(ctx, {
```

Comment:
you actually do not need to assign the new chart as a variable for this to work. It just means that you currently have a variable that's not being used for anything.